Sessions/1a-Intro-to-Python.ipynb

```diff
@@ -99,17 +209,29 @@
     "    - The programming language (e.g. Python, R, Julia, and so on)\n",
     "    - Relevant packages (e.g. pandas, matplotlib, tidyverse, ggplot)\n",
     "\n",
-    "If you are already familiar with creating Python environments, installing libraries and Jupyter Notebooks you can clone [our GitHub repo](https://github.com/Living-with-machines/dhoxss-text2tech) and use the notebooks on your machine.\n",
     "\n",
-    "### The solution!\n",
+    "## Binder\n",
     "\n",
+    "![](images/mybinder.png)\n",
     "[Binder](https://mybinder.org/) is a service that provides your code and the hardware and software to execute it.\n",
```

Comment:
I'd add an empty line after the image, so the text is easier to read (and the same for the other figures).

Reply:
Done!